minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java

```diff
@@ -277,6 +279,11 @@ public void destroy() {
             }
 
             LOGGER.info("Destroyed minimesos cluster " + clusterId);
+            try {
+                FileUtils.forceDelete(new File(getHostDir(), ".minimesos/sandbox-" + clusterId));
+            } catch (IOException e) {
+                // ignore
```

Comment:
@alex-glv, I agree with @frankscholten , `//ignore` is not enough. It's necessary to ERROR log this. 
See https://ci.infra.container-solutions.com/sonar/component_issues?id=minimesos-project#resolved=false|severities=CRITICAL on this topic